frontend_v2/src/app/components/get-involved/get-involved.component.html

```diff
@@ -1,95 +1,71 @@
 <app-header-static></app-header-static>
-<div class="container-top container align-center get-involved-container">
-  <div>
-    <h4 class="get-involved-title align-center fw-regular">Get Involved</h4>
-  </div>
-  <div class="get-involved-description">
-    <p class="fw-light content-section">
-      Thanks for your interest in helping out with the EvalAI project! We're a team of volunteers around the world who
-      want to reduce the barrier to entry for doing AI research and make it easier for researchers, students and
-      developers to develop and use state-of-the-art algorithms as a service. We are always listening for suggestions to
-      improve our platform, including identifying bugs and discussing enhancements. Here are different ways in which how
-      you can help:
-    </p>
-
-    <p></p>
-  </div>
-
-  <div class="get-involved-section">
-    <div class="row">
-      <hr />
-      <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12">
-        <div style="font-size: 24px;" class="get-involved-section-title left">Report issues</div>
-        <div class="get-involved-section-body left content">
-          <p class="fw-light content-section">
+<div class="white-bg view-container flex-body">
+  <div class="content">
+    <!-- EvalAI into -->
+    <div class="grad-container grad-vh">
+      <div class="row">
+        <div class="col s12 m8 l8 ev-mt-50">
```

Comment:
@Kajol-Kumari As we are not showing rocket container from medium sized screens then can you please also align this container to the center from that size only.

Reply:
@Sanji515 thanks for pointing this, i have Incorporated the suggested change. Can you please look at it if it's fine now?